tools/common/Driver.cs

```diff
@@ -126,6 +126,15 @@ static void AddSharedOptions (Application app, Mono.Options.OptionSet options)
 			options.Add ("root-assembly:", "Specifies any root assemblies. There must be at least one root assembly, usually the main executable.", (v) => {
 				app.RootAssemblies.Add (v);
 			});
+			options.Add ("linker-optimize=", "A comma-delimited list of optimizations to enable/disable. Example: --optimize=+remove-uithread-checks will make the linker remove the UI thread checks. Use 'all' to enable or disable all optimizations.\n" +
```

Comment:
I think the example `--optimize=` if better than the option, `linker-optimize`
It's not really linker (only related) and other, future, optimization can be even more unrelated